nss-day-cohort-48/rare-api-boastful-bulldogs
# navbar with loggedin user and admin views

**_To Test_**

`git fetch --all`
`git checkout thejmdw-navbar`

_login into the app as steve_
`me@me.com`
`me`

_**check that the navbar navigates to the correct links**_

_**log out and sign in with a user that is not staff**_
_check the categories manager and the tags manager links in the navbar_
_user should not have the option to delete or add_
